doc/geany.txt

```diff
@@ -3973,8 +3973,8 @@ On Windows 7 and above you most likely will find it at:
 ``C:\users\UserName\Roaming\geany``
 
 
-Tools menu items
-----------------
+Configuration files menu
```

Comment:
While this section is currently mainly about the `Configuration files` menu item, I think we could leave the section title as is. We might add further documentation for other `Tools` menu items here as well, so the more generic section title would serve as a container and so we could add a sub heading for `Configuration files`.

Though I don't have a strong opinion on that.